Sources/VitalHealthKit/Extensions/Date.swift

```diff
@@ -11,9 +11,9 @@ extension Date {
   
   static func dateAgo(_ date: Date = .init(), days: Int) -> Date {
     let daysAgoDate = vitalCalendar.date(byAdding: .day, value: -abs(days), to: date)
-    let beginningOfTheDay = daysAgoDate?.dayStart
+    let beginningOfTheDay = daysAgoDate!.dayStart
     
-    return beginningOfTheDay ?? date
+    return beginningOfTheDay
```

Comment:
you sure about this?

Reply:
Yes, there is no good fallout here other than crash & burn unfortunately if something such fundamental as Georgian calendar math went south 😬

And if this is the cause of the precondition failure, it is still better to have crash traces pointing here than failing somewhere else